test/verify/check-shell-multi-machine

```diff
@@ -192,6 +193,26 @@ class TestMultiMachineAdd(testlib.MachineCase):
                                     ".*: bridge failed: .*",
                                     ".*: bridge program failed: Child process exited with code .*")
 
+        # in a remote Shell secondary session, ssh-connecting to other machines happpens from primary machine
```

Comment:
In my opinion, this is a bug, not a feature.

Reply:
That's up to interpretation (bug or structural design limitation), but at least it's been the status quo for throughout Cockpit's existence. It's meant to document the currently expected behaviour.

Reply:
Yes, agreed. Still, in my opinion... :-)